### PR TITLE
Tidy and annotate CanCluster

### PR DESCRIPTION
**Pull request by Funth0mas**
- Upper case class name
- Snake case variables
- Define used instance variables in `__init__`
- Annotate

### CODE REVIEW

File: src/canmatrix/cancluster.py [hunk 1]
```diff
@@ -1,74 +1,84 @@
...
-    def ecus(self):
+    def ecus(self):  # type: () -> typing.MutableSequence[canmatrix.Ecu]
         if not self._ecus:
-            self.updateECUs()
+            self.update_ecus()
         return self._ecus
 
     @property
-    def boardUnits(self):
+    def board_units(self):  # type: () -> typing.MutableSequence[canmatrix.Ecu]
```
Comment (ebroecker) [inline src/canmatrix/cancluster.py]: ... while refactoring - pleas rename in `Ecus`

File: src/canmatrix/formats/kcd.py [hunk 2]
```diff
@@ -149,7 +149,7 @@ def dump(dbs, f, **options):
     nodeList = {}
 #    for name in dbs:
 #        db = dbs[name]
-    for bu in canClust.boardUnits:
+    for bu in canClust.board_units:
```
Comment (ebroecker) [inline src/canmatrix/formats/kcd.py]: same as above `bu` -> `ecu`

File: src/canmatrix/cancluster.py (hunk 1 above)
Comment (Funth0mas) [reply src/canmatrix/cancluster.py]: sure, thanks for hint!

**Comment by Funth0mas** [comment]
The build is failing because of missing `typing` module, it will be resolved by merging #274.

**Pull request merged**